ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -717,11 +732,8 @@ export default class ConfirmTransactionBase extends Component {
       ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
       fromInternalAccount,
       ///: END:ONLY_INCLUDE_IF
-      isSmartTransaction,
     } = this.props;
 
-    const hideLoadingIndicator = isSmartTransaction;
```

Comment:
The `isSmartTransaction` property wasn't previously defined in the container so the spinner was always shown.

This still seems desirable as it means a spinner is briefly shown before redirecting to the status confirmation.